docs/src/main/asciidoc/opentracing-guide.adoc

```diff
@@ -82,16 +102,19 @@ The first step is to start the tracing system to collect and display the capture
 docker run -e COLLECTOR_ZIPKIN_HTTP_PORT=9411 -p 5775:5775/udp -p 6831:6831/udp -p 6832:6832/udp -p 5778:5778 -p 16686:16686 -p 14268:14268 -p 9411:9411 jaegertracing/all-in-one:latest
 ----
 
-Now we are ready to run our application. Use:
+Now we are ready to run our application. If using `application.properties` to configure the tracer:
 
 [source, text]
 ----
-./mvnw compile quarkus:dev -Djvm.args="-DJAEGER_SERVICE_NAME=myservice -DJAEGER_SAMPLER_TYPE=const -DJAEGER_SAMPLER_PARAM=1 -DJAEGER_ENDPOINT=http://localhost:14268/api/traces"
+mvn compile quarkus:dev
```

Comment:
We decided to go with the Maven wrapper approach as we had a few reports with people not using the right version of Maven. Could you change it back to `./mvnw`?